department-of-veterans-affairs/va.gov-cms
# VACMS-19496: Swapping content sync for default content deploy.

## Description

Relates to #19496 

## Testing done
Local exporting and importing of a campaign landing page.

## QA steps

What needs to be checked to prove this works?
No visual or behavior changes. Ability to export and import content through drush and/or the ui.
What variations of circumstances (users, actions, values) need to be checked?
Need to check users and other content types.

### Definition of Done


### Select Team for PR review
